### DISCUSSION TITLE
supports for framework NuxtJs

### DISCUSSION THREAD

**Discussion by jusali2221** [web-infra-dev/rsbuild#1551] [Ideas]
Will there be NuxtJS support in the future?
I tried the builder on React. It's awesome, but I'm working on nuxt)

**Comment by chenjiahan** [comment]
Rspack is working on Nuxt support, see: https://github.com/web-infra-dev/rspack/issues/2592

In the short term, Rsbuild may not support Nuxt. In the long run, we hope there will be more upper-level frameworks based on Rsbuild.
